File: web/cgi-bin/cgi.py
#!/usr/bin/env python

import sys
from datetime import datetime

content: str = str(datetime.now())  # might format the output
length: int = len(content)

sys.stdout.write("HTTPS/1.1 200 OK\r\n")
sys.stdout.write("Content-Type: text/plain\r\n")
sys.stdout.write(f"Content-Length: {length}\r\n")
sys.stdout.write("\r\n")
sys.stdout.write(content)

File: web/cgi-bin/cgi.pl
#!/usr/bin/env perl

use strict;
use warnings;
use POSIX;

my $content = strftime("%Y-%m-%d %H:%M:%S", localtime);
my $length = length($content);

print <<"END";
HTTPS/1.1 200 OK\r
Content-Type: text/plain\r
Content-Length: $length\r
\r
$content
END
